src/sagas/recommended.js

```diff
@@ -268,29 +288,6 @@ function* fetchWatchingNow(uids) {
   }
 }
 
-function* fetchMissingUnits(recommendedItems) {
```

Comment:
Why did you move this?

Reply:
Want to make it public, so it will part on mdb and not recomindations